### PR TITLE
docs: add type to the code of some md files

### PR DESCRIPTION
**Pull request by FAKER-A**
Add type to the code of some md files

### CODE REVIEW

**Review by posva — CHANGES_REQUESTED** [review]
Can you remove all the changes to `docs-gitbook`? those files should be removed

**Comment by FAKER-A** [comment]
> Can you remove all the changes to `docs-gitbook`? those files should be removed

Ok Thanks Reply

File: docs/api/README.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ Since it's just a component, it works with `<transition>` and `<keep-alive>`. Wh
 
   Signature:
 
-  ```
+  ```ts
   type PositionDescriptor =
     { x: number, y: number } |
```
Comment (posva) [inline docs/api/README.md:294]:
```suggestion
    { x?: number, y?: number } |
```

**Review by posva — CHANGES_REQUESTED** [review]
The types were originally in flow, can you adapt them to TS by changing the `?{}` into `void` as well?

Thanks!

File: docs/api/README.md (hunk 1 above)
Comment (FAKER-A) [reply docs/api/README.md:294]: You mean this？
```ts
type PositionDescriptor =
    { x: number, y: number } |
    { selector: string } |
    void
```

Comment (posva) [reply docs/api/README.md:294]: yeah, that should do!

Comment (FAKER-A) [reply docs/api/README.md:294]: ok

**Review by posva — APPROVED** [review]
Thanks!

**Pull request merged**